<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWid</class>
 <widget class="QWidget" name="TableWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1,1,1">
     <item>
      <widget class="QPushButton" name="firstPageBtn">
       <property name="text">
        <string>首页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lastPageBtn">
       <property name="text">
        <string>尾页</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1">
       <item>
        <widget class="QLineEdit" name="inputPage"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalPage">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="prePageBtn">
       <property name="text">
        <string>上一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPageBtn">
       <property name="text">
        <string>下一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>导出数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
